plugins/org.jkiss.dbeaver.ui.app.standalone/plugin.xml

```diff
@@ -191,6 +196,10 @@
             </activeWhen>
         </handler>
         <handler commandId="org.eclipse.ui.edit.text.openLocalFile" class="org.jkiss.dbeaver.ui.app.standalone.actions.OpenLocalFileActionExt"/>
+        <handler
+            commandId="org.jkiss.dbeaver.core.util.collectLogs"
+            class="org.jkiss.dbeaver.ui.app.standalone.actions.CollectLogsHandler"
+        />
```

Comment:
```suggestion
            class="org.jkiss.dbeaver.ui.app.standalone.actions.CollectLogsHandler"/>
```

Reply:
Our code style permits both [adding the closing bracket to the last line of a statement](https://github.com/dbeaver/dbeaver/blob/dbeaver/dbeaver%2317003-auto-collect-log-files-from-the-ui/plugins/org.jkiss.dbeaver.cmp.simple.ui/src/org/jkiss/dbeaver/tools/compare/simple/ui/CompareObjectsPageOutput.java#L66-L68) and [putting it on its own line](https://github.com/dbeaver/dbeaver/blob/dbeaver/dbeaver%2317003-auto-collect-log-files-from-the-ui/plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java#L508-L515). I prefer the last option as I find it more readable. Also, it makes it possible to add another line to the bottom of a statement without changing the position of the closing bracket, which sometimes can make a diff clearer.

Reply:
But everything around your changes is written in the other style. It's inconsistent now here.

Reply:
Everything around my changes is written in one line and violates the 140 cap on line length.

Reply:
Oh, yeah, it's not only about the closing bracket. I'm not insisting, just don't like it.